index.html

```diff
@@ -0,0 +1,318 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I have already fixed some parts which you mentioned in the commit.

I highlighted to students in commits.

I have added emojis to my comments for the student so that he will understand more.

Thanks 👍